startup/33-plot_helper.py

```diff
@@ -0,0 +1,317 @@
+import matplotlib.pyplot as plt
+import numpy as np
+# from 32-data_export import _readable_time
+# import 31-data_analysis as da
+
+
+class open_figures():
+    def __init__(self, figure_labels):
+        for i in figure_labels:
+            plt.figure(num=i)
+
+
+class plot_uvvis(open_figures):
```

Comment:
Class name should use the PascalCase style.
```suggestion
class PlotUvvis(open_figures):
```